src/main/java/jalau/at18/katas/pokerhands/daniela/CardValuesCount.java

```diff
@@ -1,6 +1,8 @@
 package jalau.at18.katas.pokerhands.daniela;
 
+//import java.util.ArrayList;
 import java.util.HashMap;
+//import java.util.List;
```

Comment:
If no reason to keep them, lets remove commented code

Reply:
Already deleated all the comments in the code.